canonical/canonical.com
# /careers and /careers/all redesign

## Done

- Applied redesign as per [figma mock-up](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?node-id=5339%3A10871)
- Updated copy as per [copy doc](https://docs.google.com/document/d/1o8ind3Zav-kwrUDWgiMJnkw2QLpniKgdlWrw9byIvlE/edit#) 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://canonical-com-605.demos.haus/careers/ (landing page) and https://canonical-com-605.demos.haus/careers/all (roles page)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check against design and copy doc and see that everything matches
- Test search functionality
- Test filters and pagination 
- Test buttons

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/5783
